File: app/src/main/java/dev/ragnarok/fenrir/api/model/VKApiGraffiti.java
package dev.ragnarok.fenrir.api.model;

import com.google.gson.annotations.SerializedName;

public class VKApiGraffiti implements VKApiAttachment {

    @SerializedName("id")
    public int id;

    @SerializedName("owner_id")
    public int owner_id;

    @SerializedName("url")
    public String url;

    @SerializedName("width")
    public int width;

    @SerializedName("height")
    public int height;

    @SerializedName("access_key")
    public String access_key;

    @Override
    public String getType() {
        return TYPE_GRAFFITI;
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/VKApiGraffiti.kt
package dev.ragnarok.fenrir.api.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class VKApiGraffiti : VKApiAttachment {
    @SerialName("id")
    var id = 0

    @SerialName("owner_id")
    var owner_id = 0

    @SerialName("url")
    var url: String? = null

    @SerialName("width")
    var width = 0

    @SerialName("height")
    var height = 0

    @SerialName("access_key")
    var access_key: String? = null
    override fun getType(): String {
        return VKApiAttachment.TYPE_GRAFFITI
    }
}